components/Carousel/Carousel.js

```diff
@@ -33,8 +33,8 @@ class Carousel {
     this.images[this.currentIndex].classList.add("visible");
   }
 
-  changeImage(isRight) {
-    this.currentIndex += isRight ? 1 : -1;
+  changeImage(goRight) {
+    this.currentIndex += goRight ? 1 : -1;
 
     if (this.currentIndex < 0) this.currentIndex = this.images.length - 1;
     else if (this.currentIndex >= this.images.length) this.currentIndex = 0;
```

Comment:
- Jakub You did amazing job on the sprint challenge. You properly implemented Tabs and Cards components into the project and were able to filter out articles by selecting particular Tab. 
- Great job tackling the stretch goal and working on Carousel functionality. It was implemented properly. Not only that but You added transition and auto scroll to it. Amazing feature ! 🎖 🏅 
Overall:
* Tabs and Cards properly implemented (MVP)
* Good answers to questions (MVP)
* Carousel functionality implemented (Stretch)
* Animation applied (Stretch)
* Clean Code
* Fluid git workflow
Excellent job, You displayed mastery on the subject 👍